tests/unit/objects/test_projects.py

```diff
@@ -28,6 +52,106 @@ def resp_get_project():
         yield rsps
 
 
+@pytest.fixture
+def resp_user_projects():
+    with responses.RequestsMock() as rsps:
+        rsps.add(
+            method=responses.GET,
+            url="http://localhost/api/v4/users/1",
+            json=user_content,
+            content_type="application/json",
+            status=200,
+        )
```

Comment:
We can make use of the `lazy=True` argument to create a lazy object (for both user and project) without the first GET call, so we only need to mock the second call.

See docs at https://python-gitlab.readthedocs.io/en/stable/api-usage.html#lazy-objects In fact, we already have fixtures for a lot of these in unit tests for this reason:
https://github.com/python-gitlab/python-gitlab/blob/main/tests/unit/conftest.py#L62-L89 (just check that the `id` in the lazy fixture matches your next calls, otherwise you might need to tweak your reponses, but I think it's ok)

So let's pass these to the tests and we can remove the extra mocked reponses:

```suggestion
```

I realize some of the existing tests do not do that, maybe you were trying to be consistent here. So I probably need to look at cleaning up existing code as well ;)